src/operator/zip-support.ts

```diff
@@ -56,8 +56,8 @@ export class ZipSubscriber<T, R> extends Subscriber<T> {
     this.active = len;
     for (let i = 0; i < len; i++) {
       let iterator = iterators[i];
-      if (iterator.stillUnsubscribed) {
-        iterator.subscribe(iterator, i);
+      if ((iterator as any).stillUnsubscribed) {
+        (iterator as any).subscribe(iterator, i);
```

Comment:
`stillUnsubscribed` is a member of `ZipBufferIterator<,>` but not `StaticArrayIterator` or `StaticIterator`.  I could add it on as a member (just typed) then it would probably work without the cast.